src/components/form-elements/_text-input.scss

```diff
@@ -103,12 +103,18 @@ $includeHtml: false !default;
 
     &--valid {
       color: $inputFocusValidBorderColor;
-      font-weight: $fontWeightBold;
+      text-shadow: -0.2px -0.2px 0 $inputFocusValidBorderColor,
+      0.2px -0.2px 0 $inputFocusValidBorderColor,
+      -0.2px 0.2px 0 $inputFocusValidBorderColor,
+      0.2px 0.2px 0 $inputFocusValidBorderColor;
```

Comment:
Yeah, let's try `text-stroke` :) Support looks OK to me.

Reply:
changed!